### DISCUSSION TITLE
Add support for TeamCity build status message

### DISCUSSION THREAD

**Discussion by BlythMeister** [cake-build/cake#4003] [Ideas]
The current write status message is not a persistent status message (for the final build status)

Add support to update the final build status message 

Would be nice to have a prepend, append and replace method for the final build status.

Documentation of format can be found here: https://www.jetbrains.com/help/teamcity/service-messages.html#Reporting+Build+Status

As stated, the current status message can be included in the new status using `{build.status.text}`
As a convention, messages should be separated using a semi colon.

**Comment by gep13** [comment]
This has been created as an issue here: https://github.com/cake-build/cake/issues/4019 and therefore this discussion can be closed.
